.github/workflows/release.yml

```diff
@@ -107,12 +107,18 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        cpu: [x86_64, arm64]
         raytracing_label: [raytracing, no-raytracing]
+        include:
+          - cpu: x86_64
+            macos: macos-latest
+          - cpu: arm64
+            macos: macos-14
 
-    runs-on: macos-latest
+    runs-on: ${{matrix.macos}}
 
     env:
-      CMAKE_OSX_ARCHITECTURES: x86_64
+      CMAKE_OSX_ARCHITECTURES: ${{matrix.cpu}}
```

Comment:
I don't think we need this variable anymore since it's only used for cross compiling

Reply:
Maybe you are right, however we still use it in CI logic to make decisions. I'll try removing it and replacing it with an input var, but this is out of scope of this PR.